cryptus-app/components/wallet_viewer/nft_details/nft_infos/nft_buy_button/nft_buy_button.module.scss

```diff
@@ -0,0 +1,43 @@
+@use "/styles/variables" as v;
+
+.container {
+    align-self: flex-start;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    width: fit-content;
+    background-color: #eee2e5;
+    padding-left: 8px;
+    padding-right: 8px;
+    border-radius: 5px;
+}
+
+.price {
+    padding-left: 4px;
+    padding-right: 4px;
+    margin-top: -4px;
+    margin-bottom: -4px;
```

Comment:
Why is there padding 4 and margin -4? Anyways, we don't use negative margins, so find a better way to make it work.